.github/workflows/ci.yml

```diff
@@ -111,8 +111,7 @@ jobs:
       - name: Run golangci-lint
         uses: golangci/golangci-lint-action@v3.2.0
         with:
-          # Update when https://github.com/golangci/golangci-lint/issues/2997 is fixed.
```

Comment:
does `version: latest` not work anymore? I'd prefer to do that if the regression is fixed than run with an older version

Reply:
Fixed, using version: latest seems to work.